pkgs/development/interpreters/python/pypy/2.7/default.nix

```diff
@@ -15,7 +15,7 @@ let
   minorVersionSuffix = "";
   pythonVersion = "2.7";
   version = "${majorVersion}.${minorVersion}${minorVersionSuffix}";
-  libPrefix = "pypy${majorVersion}";
+  libPrefix = "pypy${pythonVersion}";
   sitePackages = "site-packages";
```

Comment:
While at it:
```
sitePackages = "lib/${libPrefix}/site-packages";
```

Reply:
That doesn’t work (see main comment thread).